panel/panel.py

```diff
@@ -118,9 +97,22 @@ def ensure_panel(window: sublime.Window):
     return get_panel(window) or create_panel(window)
 
 
-def filter_errors(window, errors):
-    vids = [v.id() for v in window.views()]
-    return {vid: d for vid, d in errors.items() if vid in vids}
+def sort_errors(errors):
+    return sorted(errors, key=lambda x: (x["line"], x["start"], x["end"]))
+
+
+def get_filter_and_sort(buf_errors, panel_filter):
```

Comment:
get_filter_and_sort -> filter_and_sort otherwise in indicates you return probably a function

Reply:
Absolutely. That function names is bad.